script.js

```diff
@@ -10,3 +10,191 @@ document.querySelectorAll('.mobile-link').forEach((n) => n.addEventListener('cli
   hamburger.classList.remove('active');
   mobileMenu.classList.remove('active');
 }));
+
+// Navbar functions end here -----------------------^
+
+// PopScreens goes there------------------------------------->
+
+const popWindow1 = document.getElementById('popup-window');
+
+const popData = [
+  {
+    title: 'Keeping track of hundreds of components',
+    technologies: ['Ruby on rails', 'css', 'Javascript'],
+    technologies2: ['Codekit', 'GitHub', 'JavScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    btn1: 'See Live',
+    btn2: 'See Source',
+    image: '/portfolio-Img/mobile-popup.png',
+    linkToLiveSource: '#',
+    linkToSource: '#',
+  },
+];
+
+function popDisplay() {
+  const popScreen = `<div class="popup-2">
+                    <img data-close-button class="desktop-close" src="./portfolio-Img/Disabled.png" alt="popup close icon">
+                    <div class="mobile-popup-img">
+                    <img src= ${popData[0].image} alt="cover-picture-for-mobile-popup" class="cover">
+                    <img data-close-button class ="close" src="./portfolio-Img/Icon - Cancel.png" alt="popup close icon">
+                    </div>
+                    <div class="desktop-popup-buttons">
+                        <button class="desk-btn-1">
+                          <div class="popup-btn-text">${popData[0].btn1}</div>
+                          <div class="popup-icons">
+                            <img src="portfolio-Img/see live icon.png" alt="see-live-icon">
+                          </div>
+                        </button>
+                        <button class="desk-btn-2">
+                        <div class="popup-btn-text">${popData[0].btn2}</div>
```

Comment:
- [ ] Instead of static data for your popups, kindly consider generating popups dynamically with the same cardData array. that way in the future you only need to add new projects in your cardData array and then all of your cards and their popup modal will be created dynamically from single source of data.
  -  Modal popup data is populated with the data from the projects array.